scripts/download_model.py

```diff
@@ -1,5 +1,13 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "transformers",
+#     "torch",  # required by transformers for model handling
+# ]
+# ///
+
 from transformers import AutoModel, AutoTokenizer
 
 # Model will download the first time they are initialized
-tokenizer = AutoTokenizer.from_pretrained("BAAI/bge-large-en-v1.5")
-model = AutoModel.from_pretrained("BAAI/bge-large-en-v1.5")
+tokenizer = AutoTokenizer.from_pretrained("BAAI/bge-small-en-v1.5")
+model = AutoModel.from_pretrained("BAAI/bge-small-en-v1.5")
```

Comment:
no downgrade in performance? My gut feeling says there shouldn't be anything very noticable.

Reply:
ah just read "results between two felt comparable". Nice

Reply:
yep, it was close enough from my testing that it was worth it for the time/mem/disk savings.